scripting/get5.sp

```diff
@@ -847,7 +847,10 @@ public Action Command_Stop(int client, int args) {
     }
 
     int roundsPlayed = GameRules_GetProp("m_totalRoundsPlayed");
-    if (g_GameState != GameState_Live || roundsPlayed == 0) {
+    int roundsPerHalf = GetCvarIntSafe("mp_maxrounds") / 2;
+
+    // ( Match Not Live || No Rounds Played || Half Time || Full Time, 2 * Half Time)
+    if (g_GameState != GameState_Live || roundsPlayed == 0 || roundsPlayed == roundsPerHalf || (roundsPlayed == (2 * roundsPerHalf))) {
```

Comment:
I would think just checking g_PendingSideSwap would be sufficient here (in addition to the original checks).